Makefile

```diff
@@ -41,7 +41,7 @@ generate: build # Generate CLI docs and samples
 	go run ./hack/generate/samples/generate_testdata.go
 
 .PHONY: bindata
-OLM_VERSIONS = 0.16.1 0.15.1
+OLM_VERSIONS = 0.17.0 0.16.1 0.15.1
```

Comment:
Well, we need not add modifications in the script if we specify the right tag here. ie: `OLM_VERSIONS = v0.17.0 0.16.1 0.15.1`